main.scss

```diff
@@ -9,17 +9,19 @@
 @import 'src/scss/variables';
 
 /// Outputs all features and styles available for o-footer-services.
-/// @param {map} $opts [('logo': 'ftlogo-v1:origami', 'icons': ('slack', 'github'))] - A map of optional footer features to output, including the footer logo and list of social icon links.
+/// @param {map} $opts [('logo': 'ftlogo-v1:origami', 'icons': ('slack', 'github'), 'brand-strip': true)] - A map of optional footer features to output, including the footer logo, list of social icon links, and "a Nikkei company" brand strip.
 /// @param {bool} $include-base-styles [true] - Whether to output fundamental o-footer-services styles required by the optional features.
 /// @access public
 @mixin oFooterServices($opts: (
 	'logo': false,
 	'icons': ('slack', 'github'),
+	'brand-strip': true,
```

Comment:
I think making this true by default would be a breaking change because the new mixin can throw an error if no $system-code variable is set -- https://github.com/Financial-Times/o-footer-services/pull/75/files#diff-69f6b0dfb3c2d9595a229fe49a5891ce3354348b3fb4db9234d2fc2e4c27b052R196

What do you think @notlee?

Reply:
Good shout! That could have been annoying (we want it to be available by default because passing no options should give you all the css you need to use a component). Fortunately, there's already a check for the system code variable at the top of `oFooterServices` so I can remove this